### ISSUE TITLE
Failed to load plugin 'tailwindcss' declared in '.eslintrc#overrides[0]': Unexpected token '?'

### ISSUE THREAD

**Issue by szygendaborys** [ixartz/Next-js-Boilerplate#36]
Hi!

I've experienced a problem while bootstrapping this project.

The way to reproduce this error is to bootstrap a fresh version from this repo by using `git clone`. Install npm dependencies and try to execute `npm run lint`.

This occurs in such error:
```
Failed to load plugin 'tailwindcss' declared in '.eslintrc#overrides[0]': Unexpected token '?'
```

Do you have any clues on how to resolve this problem?

**Comment by ixartz** [comment]
Hi @szygendaborys,

Thank you for sharing your issue.

I've made a clean and fresh clone of the project using `git clone` and I'm not able to reproduce your issue.

Here is the step I have followed:
- git clone git@github.com:ixartz/Next-js-Boilerplate.git
- npm install
- npm run install

Here is the result:
```sh
npm run lint

> next-js-boilerplate@1.1.0 lint
> next lint

✔ No ESLint warnings or errors
```

So on my computer, everything is working fine.
